webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3587,7 +3584,93 @@ var userOverride = {
                         $applozic('.mck-dropup-menu').hide();
                 }
             };
-
+            _this.triggerWelcomeEvent = function(){
+                window.Applozic.ALApiService.sendMessage({
+                    data: {
+                        message: {
+                            type: 5,
+                            contentType: 10,
+                            message: 'Event: WELCOME',
+                            groupId: CURRENT_GROUP_DATA.tabId,
+                            metadata: {
+                                category: 'HIDDEN',
+                                KM_TRIGGER_EVENT: 'WELCOME',
+                            },
+                            source: 1,
```

Comment:
In case of dialogflow, what will happen if someone doesn't have 'WELCOME' event as default welcome intent, instead he has different intent as welcome intent.

Reply:
For now, it will only trigger the Welcome Message.

@Ojashwa We can file an improvement issue for this use case. Let users change the intent they want to trigger on 'Restart conversation'.

CC: @devashishmamgain

Reply:
it will be easy change to implement, add another property in defaultSettings for welcome event name https://docs.kommunicate.io/docs/web-conversation#conversation-settings
if customer configure it then use that, else fallback to the default 'WELCOME'